cardinal/config_test.go

```diff
@@ -266,3 +270,84 @@ func defaultConfigWithOverrides(overrideCfg WorldConfig) WorldConfig {
 
 	return cfg
 }
+
+func TestWorldConfig_loadWorldConfigUsingTomlEnv(t *testing.T) {
+	tomlFilePath := "../e2e/testgames/world.toml"
+	t.Setenv(tomlConfigFileEnvVariable, tomlFilePath)
+
+	cfg, err := loadWorldConfig()
+	assert.NilError(t, err)
+	assert.Equal(t, "my-world-e2e", cfg.CardinalNamespace)
+}
+
+func TestWorldConfig_loadWorldConfigUsingTomlFlag(t *testing.T) {
+	// Set up command-line arguments
+	os.Args = []string{"cmd", "--TOML_CONFIG_FILE=../e2e/testgames/world.toml"}
```

Comment:
didn't we change this to `CARDINAL_CONFIG`? i wonder why this test still passes

Reply:
ahh, it's because viper value still there after other test case execution. need to reset the viper value after running the test cases.